patches/minecraft/net/minecraft/client/renderer/LightTexture.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/client/renderer/LightTexture.java
 +++ b/net/minecraft/client/renderer/LightTexture.java
+@@ -122,7 +122,7 @@
+                         vector3f1.func_229190_a_(vector3f3, f9);
+                      }
+                   }
+-
++                  net.minecraftforge.client.ForgeHooksClient.onDimensionLightMapUpdate(clientworld, p_205106_1_, f, f4, f5, vector3f1);
```

Comment:
Minor nitpick: Could you add a blank line above your change, so the `-` diff is removed? (just a minor nitpick tho)

Reply:
Sure, thanks!